model/src/main/kotlin/OrtResult.kt

```diff
@@ -342,6 +342,7 @@ data class OrtResult(
      * from a VCS the analyzer root is the root of the working tree, if the project was not checked out from a VCS the
      * analyzer root is the input directory of the analyzer.
      */
+    @Suppress("UNUSED") // This is intended to be mostly used via scripting.
```

Comment:
Hmm, I'm not sure this is semantically the right issue to be suppressed here: The function *is* used in the main ORT code after all. Compare to e.g. `getUncuratedPackageById`, if you comment out the `@Suppress("UNUSED")` there, the function gets grayed out and displays a hint that it "is never used". That is not the case for the `getFilePathRelativeToAnalyzerRoot` function, and thus I don't think we should create the impression of it being unused by suppressing that issue, if in reality it is not unused.

Reply:
An idea how to solve that would be to create a new file with OrtResult extension / script helper functions, and suppress `UNUSED` for that whole file. That would clean-up the huge `OrtResult` class, which is also nice, and I believe was suggested by @mnonnenmacher also before.

I can take that task, if you want @fviernau.

Reply:
Yeah, agree that this is not 100% correct and I'm fine not doing this commit as is. There are other functions as well which are annotated similar and used from within ORT, e.g. see ```getDetectedLicensesForId()```.

(Assuming that it is ok to use it via scripting, we currently do, but maybe we shouldn't:)
I wonder how we could best indicate that something is intended to be used via scripting.
Do you have any idea?

Reply:
See my second comment above, which probably overlapped with yours.

Reply:
@sschuberth good idea! 
I'm currently cleaning things up in OrtResult anyway so I prefer to do this task myself if that's ok with you.

anyhow my gut feeling is that the timing to do it now now is not best, e.g. 
* things in OrtResult will clarify and clean-up further
* with that it may become clearer if and how exactly that should be done
* imo cleaning things up can be nicely done when touching it, not separately

An alternative idea how to address this could be to add a custom annotation ```@OrtScriptingApi``` which e.g. inherits @Suppress(). What do you think about that?

Reply:
> so I prefer to do this task myself if that's ok with you.

:+1: 

> What do you think about that?

Honestly, I don't think it's worth it. I'd prefer to simply not address this at all until we move this code to a new file.